docs/standard/library-guidance/cross-platform-targeting.md

```diff
@@ -1,19 +1,26 @@
 ---
 title: Cross-platform targeting for .NET libraries
 description: Best practice recommendations for creating cross-platform .NET libraries.
-ms.date: 11/11/2024
+ms.date: 08/20/2025
 ---
 
 # Cross-platform targeting
 
 Modern .NET supports multiple operating systems and devices. It's important for .NET open-source libraries to support as many developers as possible, whether they're building an ASP.NET website hosted in Azure, or a .NET game in Unity.
 
+When choosing target frameworks for your library, it's important to distinguish between two different goals:
+
+- **Cross-platform compatibility**: Your library works on different operating systems (Windows, Linux, macOS). Modern .NET versions like .NET 8+ provide excellent cross-platform support.
```

Comment:
```suggestion
- **Modern APIs and patterns**: Your library leans into the benefits of the recent versions of the .NET platform. You use modern language patterns and support advanced deployment models like Native Ahead-of-Time compilation on the CoreCLR.
```

Cross-platform is the wrong emphasis here - all TFMs are reasonable in that regard. Instead, newer TFMs let you lean in to more modern features of the .NET platform itself, not necessarily _the target OS platform_.